package.json

```diff
@@ -72,7 +72,9 @@
     "modals": "^0.1.5",
     "scroll-to": "0.0.2",
     "semver": "^5.1.0",
-    "weex-components": "^0.1.3"
+    "transitionize": "0.0.3",
```

Comment:
Why added `transitionize` into dependencies? And where it would be used to?
Thanks.